AferoSwiftSDK/Core/Models/OfflineSchedule.swift

```diff
@@ -841,14 +847,103 @@ open class OfflineSchedule: NSObject {
             public typealias Hour = UInt8
             public typealias Minute = UInt8
             
-            public var dayOfWeek: DayOfWeek
-
+            private var _daysOfWeek: Set<DayOfWeek> = []
+
+            public var useCompactDayRepresentation: Bool = false
+
+            private static let compactFlag: UInt8 = 128;
```

Comment:
Generally good with this. Couple of potential improvements:
* We may want to consider is attaching the bit indexes to `DayOfWeek` (via a extension).
* Even if we don't attach to DayOfWeek, these should be an enum with associated type `Uint8`.
* Rather than using the actual integer values here, we may want to consider using the bit indexes (which are also the powers of 2). This will make the structure in which these are packed a little more clear, and to set we could just do a `dayByte |= 2**dayOfWeek.bitIndex` (that assumes we're hanging the bit index off of DayOfWeek as described in the previous bullet).
* Are the bit indexes equal to the associated day of week ordinals? Can we reuse this and reduce the amount of code?